documentation/guides/generating-components.md

```diff
@@ -28,10 +28,12 @@ In response to the above, the component name should be the dashified version of
    package.json
 ```
 
-From here, peek into the `package.json` file and ensure the "devDependency" of `@spectrum-css/spectrumpattern` listed there is of the most current version. Once you have confirmed that run `yarn` in your terminal to get any new packages added to the repository and have the CSS provided therein processed for use in your component using the default configuration. You'll now be able to review your component via the Storybook command `yarn storybook` or see it appear in tests via `yarn test` so that you can dive into fully preparing the patterns for use as a custom element.
+From here, peek into the `package.json` file and ensure the "devDependency" of `@spectrum-css/spectrumpattern` listed there is of the most current version. Then, run `yarn` in your terminal to grab any newly-added packages, as well as to ensure that you have the provided CSS processed for use in your component. You'll now be able to see your component in the Storybook, using the command `yarn storybook`, or test its functionality via `yarn test` so that you can dive into fully preparing the pattern for use as a custom element.
```

Comment:
Everything through
```
From here, peek into the `package.json` file and ensure the "devDependency" of `@spectrum-css/spectrumpattern` listed there is of the most current version. Then, run `yarn` in your terminal to grab any newly-added packages, as well as to ensure that you have the provided CSS processed for use in your component.
```
has changed a little bit.

Do you want to checkout the workflow as it is now and edit this PR, or move this one along and create a follow up that updates the workflow of this `yarn new-package` command?

Reply:
Hmm. I might just push this one for now and create a follow-up PR tomorrow. Thank you for letting me know, I figured there'd be some deprecated stuff in here :)